wordcloud/wordcloud.py

```diff
@@ -503,16 +508,17 @@ def process_text(self, text):
 
         flags = (re.UNICODE if sys.version < '3' and type(text) is unicode
                  else 0)
-        regexp = self.regexp if self.regexp is not None else r"\w[\w']+"
+        regexp = self.regexp if self.regexp is not None else r"(\w[\w']+|\w+)"
```

Comment:
if you add another "if" that requires another option, so I'd rather avoid it. The regex is very flexible and I'd like to include much of the options there.
We could actually try changing the regexp for digits and not use the ``isdigit`` function maybe?